packages/esm-form-render-app/src/render/forms-render-test.component.tsx

```diff
@@ -74,6 +75,65 @@ function FormRenderTest() {
     setIsSchemaLoaded(false);
   };
 
+  const formValidator = () => {
+    if (defaultJson) {
+      const parsedForm = typeof defaultJson == 'string' ? JSON.parse(defaultJson) : defaultJson;
+
+      for (let i = 0; i < parsedForm.pages.length; i++) {
+        for (let j = 0; j < parsedForm.pages[i].sections.length; j++) {
+          for (let k = 0; k < parsedForm.pages[i].sections[j].questions.length; k++) {
+            
+            const questionObject = parsedForm.pages[i].sections[j].questions[k];
+            handleFormValidation(questionObject);
+          }
+        }
+      }
+    } else {
+      console.log('Empty form!');
+    }
+  };
+
+  const handleFormValidation = (conceptObject) => {
+    conceptObject.questionOptions.concept
+      ? openmrsFetch(`/ws/rest/v1/concept/${conceptObject.questionOptions.concept}`)
+          .then((response) => {
+            console.log(response.data);
+            dataTypeChecker(conceptObject, response);
+          })
+          .catch((error) => {
+            error.message.includes('404') &&
```

Comment:
I can add some configuration to check for 500 errors but I don't think they can be caught in the catch block.

Reply:
I stand corrected, couldn't really test for server errors but incase a 500 error occurs in the error message it will be logged. Also added catcher for 400, 401 and 403 errors.